src/index.d.ts

```diff
@@ -0,0 +1,13 @@
+import pretty from 'pino-pretty'
+export interface Request {
+    method: string;
+    url: string;
+}
+export declare type LogDescriptor = Record<string, unknown> & {
+    time: number;
+    level: number;
+    req?: Request;
+};
+export declare const messageFormatFactory: (colorize: boolean) => (log: LogDescriptor, messageKey: string) => string
```

Comment:
I noticed `colorizerFactory` is exported in the `pino-pretty` js file but not in the ts definition file.

Do we no want to expose the `messageFormatFactory`, `Request` & `LogDescriptor` too? But only export `target`.

Reply:
> I noticed colorizerFactory is exported in the pino-pretty js file but not in the ts definition file.

PR to pino-pretty?